### DISCUSSION TITLE
Run/install local plugin

### DISCUSSION THREAD

**Discussion by orrosman** [Flow-Launcher/Flow.Launcher#981] [Q&A] [answered]
Hi all,
I intend to try and develop a plugin, and couldn't find in the docs any reference to running a locally developed plugin.
I want to run the plugin locally to test it before any PR (obviously).

Have I missed something in the docs?
Thanks for the help🙏

**Comment by Garulf** [comment, marked as answer]
You can run your plugin locally by making sure a proper `plugin.json` file in in the root of the plugins own directory. More info here:

https://www.flowlauncher.com/docs/#/plugin.json

Also the plugin needs to be in Flow's plugin folder. 

You may need to restart Flow in order for it to register.

**Reply by orrosman** [reply]
Thanks for the quick reply!
